pullrequest.rst

```diff
@@ -148,6 +151,13 @@ list anything), you will want to push your branch to your fork::
 
 This will get your changes up to GitHub.
 
+.. seealso::
+
+   For more information about git commands, please refer to:
+
+   * `Git Cheat Sheet <https://services.github.com/on-demand/downloads/github-git-cheat-sheet.pdf>`_
+   * `ProGit <https://git-scm.com/book/en/v2>`_
+
 Now you want to
 `create a pull request from your fork <https://help.github.com/articles/creating-a-pull-request-from-a-fork/>`_.
 If this is a pull request in response to a pre-existing issue on the
```

Comment:
Yes, it should be moved earlier, and mention that creating an issue is not necessary for trivial changes.